### ISSUE TITLE
Review the usage of RichText multiline / aria-multiline

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6005]
In ARIA terms, `role="textbox"` and `aria-multiline="true"` should be used to give elements the semantics of an input field and textarea.

- `role="textbox"` + `aria-multiline="false"` (or omitted) -> input field
- `role="textbox"` + `aria-multiline="true"` -> textarea

In Gutenberg, the `aria-multiline` attribute is used inconsistently. Paragraphs, for example, are not multiline. However, we'd want them to be announced as textareas, not as input fields.

This is how VoiceOver announces a paragraph right now: "text field":

<img width="738" alt="screen shot 2018-04-05 at 12 51 11" src="https://user-images.githubusercontent.com/1682452/38362383-f69c62d0-38d0-11e8-8ed5-daf181e58eae.png">

Setting `aria-multiline` to `true` makes VoiceOver announce it as a textarea:

<img width="689" alt="screen shot 2018-04-05 at 12 52 24" src="https://user-images.githubusercontent.com/1682452/38362418-1698a0b2-38d1-11e8-9b52-571c447181bf.png">

I'm not sure if the `multiline` prop on `RichText` has other purposes, but we should use `role="textbox"` + `aria-multiline="true"` on all the blocks `RichText` we want to be announced as tetareas.

**Comment by afercia** [comment]
Fixed in https://github.com/WordPress/gutenberg/pull/7306 /Cc @jorgefilipecosta
